test/e2e/kube-ovn/pod/pod.go

```diff
@@ -103,4 +122,182 @@ var _ = framework.Describe("[group:pod]", func() {
 			framework.ExpectContainElement(actualRoutes, r)
 		}
 	})
+
+	framework.ConformanceIt("should support http and tcp liveness probe and readiness probe in custom vpc pod ", func() {
+		f.SkipVersionPriorTo(1, 12, "This feature was introduced in v1.12")
+
+		daemonSetClient := f.DaemonSetClientNS(framework.KubeOvnNamespace)
+		originDs := daemonSetClient.Get("kube-ovn-cni")
+		modifyDs := originDs.DeepCopy()
+
+		newArgs := originDs.Spec.Template.Spec.Containers[0].Args
+		for index, arg := range newArgs {
+			if arg == "--enable-tproxy=false" {
+				newArgs = append(newArgs[:index], newArgs[index+1:]...)
+			}
+		}
+		newArgs = append(newArgs, "--enable-tproxy=true")
+		modifyDs.Spec.Template.Spec.Containers[0].Args = newArgs
+
+		daemonSetClient.PatchSync(modifyDs)
+
+		custVPCSubnetName := "subnet-" + framework.RandomSuffix()
+		extraSubnetNames = append(extraSubnetNames, custVPCSubnetName)
+
+		ginkgo.By("Create Custom Vpc subnet Pod")
+		vpcName = "vpc-" + framework.RandomSuffix()
+		customVPC := framework.MakeVpc(vpcName, "", false, false, nil)
+		vpcClient.CreateSync(customVPC)
+
+		ginkgo.By("Creating subnet " + custVPCSubnetName)
+		cidr = framework.RandomCIDR(f.ClusterIpFamily)
+		subnet := framework.MakeSubnet(custVPCSubnetName, "", cidr, "", vpcName, "", nil, nil, nil)
+		_ = subnetClient.CreateSync(subnet)
+
+		ginkgo.By("Creating pod with HTTP liveness and readiness probe that port is accessible " + podName)
+		pod := framework.MakePod(namespaceName, podName, nil, map[string]string{util.LogicalSwitchAnnotation: custVPCSubnetName}, framework.NginxImage, nil, nil)
+
+		pod.Spec.Containers[0].ReadinessProbe = &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Port: intstr.FromInt(80),
+				},
+			},
+			InitialDelaySeconds: 15,
```

Comment:
```suggestion
```
Is the initial delay necessary?

Reply:
not necessary

Reply:
If the delay is not necessary, we should remove it to reduce the time spent on the test case.